orchestra/static/orchestra/team-info/team-info-card.html

```diff
@@ -1,4 +1,7 @@
 <section class="section-panel todo-list">
+  <div class="overlay" ng-if="teamInfoCard.loading">
```

Comment:
sanity-checking that this works

Reply:
https://www.loom.com/share/2cf3adb9ec9d491fa4e8b553c261e4ef